README.md

```diff
@@ -1,6 +1,8 @@
 tree-sitter-java
 ================
 
-[![Build/test](https://github.com/tree-sitter/tree-sitter-java/actions/workflows/ci.yml/badge.svg)](https://github.com/tree-sitter/tree-sitter-java/actions/workflows/ci.yml)
+[![CI](https://github.com/tree-sitter/tree-sitter-java/actions/workflows/ci.yml/badge.svg)](https://github.com/tree-sitter/tree-sitter-java/actions/workflows/ci.yml)
+[![Discord](https://img.shields.io/discord/1063097320771698699?logo=discord)](https://discord.gg/w7nTvsVJhm)
```

Comment:
This PR add advertisement for this discord server centered on tree-sitter.
I don't mind either way, but I want to make sure folks are fine about it

Reply:
It is pinned on the main tree-sitter repo's discussions